xskillscore/tests/test_contingency.py

```diff
@@ -116,9 +88,124 @@ def logical(ds, edges):
         ("gerrity_score", (2 * (3 / 4) + 1 * -1 + 2 * -1 + 2 * (4 / 3)) / 7),
     ],
 )
-def test_dichotomous_scores(dichotomous_Contingency, method, expected):
+def test_dichotomous_scores(dichotomous_Contingency_1d, method, expected):
     """Test score for simple 2x2 contingency table against hand-computed values
     Scores are for H/TP: 2, M/FN: 2, FA/FP: 1, CN/TN: 2
     """
-    xs_score = getattr(dichotomous_Contingency, method)().item()
+    xs_score = getattr(dichotomous_Contingency_1d, method)().item()
     npt.assert_almost_equal(xs_score, expected)
+
+
```

Comment:
It looks like all the ROC tests are on 1D data. Could we add a multi-dimensional test?